src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/LgtmScore.java

```diff
@@ -49,11 +54,19 @@ public ScoreValue calculate(Value... values) {
     Value<LgtmGrade> worseLgtmGrade = findValue(values, WORSE_LGTM_GRADE,
         "Hey! You have to tell me the worse LGTM grade for the project!");
 
+    if (usesLgtm.isUnknown() && !worseLgtmGrade.isUnknown()) {
+      throw new IllegalArgumentException(
+          "Hey! It's unknown if the project uses LGTM but then you gave me the worse grade!");
```

Comment:
Please change the end of the sentence to "gave me the worst grade!"

Reply:
Then, I think "worse" needs to be replaced with "worst" everywhere.